src/app/services/firebase/firebase-google-auth.service.ts

```diff
@@ -10,8 +11,19 @@ export class FirebaseGoogleAuthService {
 
   constructor(private nvc: NavController) { }
 
-  doAuth(){
-    var provider = new firebase.auth.GoogleAuthProvider();
-    return firebase.auth().signInWithPopup(provider)
+  async doAuth() {
+    try {
+      const gplus = await GooglePlus.login({
+        'webClientId': '874703078872-qvo97ne2a39iski4i5gsf0vd0be1q5k4.apps.googleusercontent.com', //id do we client em credentials
```

Comment:
Please, put de webClientId value in a external file and share here and package.json (WEB_APPLICATION_CLIENT_ID). I believe it is the same value.